code/modules/antagonists/abductor/machinery/experiment.dm

```diff
@@ -112,7 +112,7 @@
  * * type The type of experiment to be performed
  * * user The mob starting the experiment
  */
-/obj/machinery/abductor/experiment/proc/experiment(mob/occupant, type, mob/user)
+/obj/machinery/abductor/experiment/proc/experiment(mob/occupant, type, mob/user, custom_objective)
 	LAZYINITLIST(history)
 	var/mob/living/carbon/human/H = occupant
```

Comment:
put `if(OOC_FILTER_CHECK(custom_objective))` here and nullify the message and send admin report if it's caught.